src/components/comment-input.component.js

```diff
@@ -113,12 +120,12 @@ export class CommentInput extends Component {
                   : translate('issue.main.commentInput', language)
               }
               multiline
-              blurOnSubmit
+              blurOnSubmit={false}
               onChangeText={text => this.setState({ text })}
               onContentSizeChange={event =>
                 this.setState({ height: event.nativeEvent.contentSize.height })}
               onSubmitEditing={event =>
```

Comment:
Yes, it's sent ... and wait .. After all, we have a comment sent only by [the "Post" button](https://github.com/lex111/git-point/blob/d1668b4e0cd1e003554991efaa0109cd30caa0c4/src/components/comment-input.component.js#L146), just there is a send handler.